plugin.xml

```diff
@@ -22,10 +23,10 @@
         <param name="android-package" value="myjni.OpenSmile"/>
       </feature>
     </config-file>
-	<config-file target="AndroidManifest.xml" parent="/manifest">
+    <config-file target="AndroidManifest.xml" parent="/manifest">
         <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
What is the `INTERNET` permission used for?

Reply:
The former developers were gone and did not reply to my email regarding these changes. Could you please simply have a trail by testing what happens when this permission is commented out? Thanks a million.